lte/gateway/c/core/oai/test/amf/test_amf_encode_decode.cpp

```diff
@@ -928,6 +928,56 @@ TEST(test_optional_dnn_pdu, test_pdu_session_establish_optional) {
   bdestroy(buffer);
 }
 
+TEST(test_optional_dnn_dotted_pdu, test_pdu_session_establish_optional) {
+  uint32_t bytes         = 0;
+  uint32_t container_len = 0;
+  bstring buffer;
+  amf_nas_message_t msg = {};
+
+  // build uplinknastransport
+  // uplink nas transport(pdu session request)
```

Comment:
remove if unused

Reply:
Ulas,
Added this test case for UT of dotted dnn encoding and decoding

Reply:
removed unused code